config/config.yaml

```
@@ -129,6 +129,10 @@ storage:
     # If 0 - no optimization threads, optimizations will be disabled.
     max_optimization_threads: null
 
+  # This section has the same options as 'optimizers' above. All values specified here will overwrite the collections
+  # optimizers configs regardless of the config above and the options specified at collection creation.
+  optimizers_overwrite: null
```

```suggestion
  optimizers_overwrite: {}
```

Can we do this instead as empty object, with no changed parameters?

---

Just curious but what exactly is different compared to `null`?

---

It's subtle.

To me `null` looks like you'd specify a single value in that field. While `{}` hints it being an object in which you can specify multiple parameters.

---

Maybe it would be even better to do it like this:

```suggestion
  # optimizers_overwrite:
  #   indexing_threshold_kb: 20000
  #   max_optimization_threads: null
```

What do you think?

---

Giving it a second thought, it's indeed easier to set parameters when you just have to uncomment them, so I think it's a good idea!